INTITA/Modules/LogInScreen/LogInViewController.swift

```diff
@@ -8,10 +8,11 @@
 import UIKit
 
 enum LoginCells: Int {
-    case logoImageCell = 0
-    case emptyCellOne
-    case emailTextFieldCell
+    case emptyCellOne = 0
```

Comment:
names for cells and their raw values are confusing

Reply:
I have just resolved it.